orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/manifest/FindManifestStage.java

```diff
@@ -17,21 +17,21 @@
 
 package com.netflix.spinnaker.orca.clouddriver.pipeline.manifest;
 
-import com.netflix.spinnaker.orca.clouddriver.tasks.artifacts.FindArtifactsFromResourceTask;
+import com.netflix.spinnaker.orca.clouddriver.tasks.artifacts.FindManifestTask;
 import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder;
 import com.netflix.spinnaker.orca.pipeline.TaskNode;
 import com.netflix.spinnaker.orca.pipeline.model.Stage;
 import com.netflix.spinnaker.orca.pipeline.tasks.artifacts.BindProducedArtifactsTask;
 import org.springframework.stereotype.Component;
 
 @Component
-public class FindArtifactsFromResourceStage implements StageDefinitionBuilder {
-  public static final String PIPELINE_CONFIG_TYPE = "findArtifactFromResource";
+public class FindManifestStage implements StageDefinitionBuilder {
```

Comment:
@lwander are we ok with breaking backwards compatibility or do we need to keep the old stage and name around?

Reply:
I've once renamed a stage and it caused a lot of problems in the execution repository -- I think we need to keep the name around.

Reply:
Alright, can just keep the old name then.